Gemfile

```diff
@@ -10,7 +10,7 @@ group(:deployment, :development) do
   gem("rake")
 end
 
-gem("bundler", "~> 1.17")
+gem("bundler", ">= 2")
```

Comment:
Should not bundler be in the gemspec since it is a runtime dependency? Can tapioca run without bundler being installed?

Reply:
Ooops, that's correct. Fixing it now.